### DISCUSSION TITLE
Akka.NET Community Standup - March 17 2021

### DISCUSSION THREAD

**Discussion by Aaronontheweb** [akkadotnet/akka.net#4856] [Community Standups]
Going live at 1:30 CST today!

# Live Stream / Recording 

https://www.youtube.com/watch?v=dUlE0Ij81to

# Agenda

- Akka.NET v1.4.18

1. Akka.DependencyInjection improvements
2. Props updates / changes
3. Anything else?

- Community Standards
- Akka.Persistence Updates

1. - Redis Overhaul
2. - Postgresql Overhaul
3. - Linq2Db Updates

- Artery & Akka.NET v1.5 Updates
- Other Community Updates

1. - Aaron Akka.NET Collaborative Editing Series

**Comment by to11mtm** [comment]
Akka.Persistence.Linq2db Repo: https://github.com/akkadotnet/Akka.Persistence.Linq2Db/ . Please refer to the readme for instructions on configuration, if there are any questions please [create an issue](https://github.com/akkadotnet/Akka.Persistence.Linq2Db/issues/new)

Nuget Package: https://www.nuget.org/packages/Akka.Persistence.Sql.Linq2Db/0.90.1
